images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -487,6 +487,11 @@ function Get-PowershellVersion {
     return $powershellVersion
 }
 
+function Get-SwigVersion {
+    $swigVersion = Run-Command "swig -version"
```

Comment:
This returns not the version, but the full output
![image](https://user-images.githubusercontent.com/48208649/120444611-73c61380-c390-11eb-9e09-c0f4412753a5.png)

Reply:
Fixed

Reply:
fixed